src/fragments/lib/in-app-messaging/js/create-campaigns.mdx

```diff
@@ -0,0 +1,62 @@
+A campaign is a messaging initiative that engages a specific audience [segment](https://docs.aws.amazon.com/pinpoint/latest/userguide/segments.html). A campaign sends tailored messages according to a schedule that you define. You can use the AWS console to create a campaign that sends messages through any single channel that is supported by Amazon Pinpoint: mobile push, In-App, email, SMS or custom channels. [Learn how to create a campaign using Amazon Pinpoint](https://docs.aws.amazon.com/pinpoint/latest/userguide/campaigns.html) to continue integrating in-app messages in your app with Amplify.
+
+1. Login to the [AWS Console](https://console.aws.amazon.com/console/home), and Search for “Amazon Pinpoint”.
```

Comment:
Nit: With all these quoted call to action items, I think we should be consistent and **bold** them instead..
```suggestion
1. Login to the [AWS Console](https://console.aws.amazon.com/console/home), and Search for **Amazon Pinpoint**.
```